### PR TITLE
Update in the max number of segments for IB

### PR DESCRIPTION
**Pull request by anwara-ms**
Update in the max number of segments for IB

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/information-barriers-in-teams.md [hunk 1]
```diff
@@ -197,6 +197,8 @@ For more information on licenses and permissions, including plans and pricing, s
 
 - **Users can't join channel meetings**: If IB policies are enabled, users aren't allowed to join channel meetings if they're not a member of the team. The root cause is that IB checks rely on whether users can be added to a meeting chat roster, and only when they can be added to the roster are they allowed to join the meeting. The chat thread in a channel meeting is available to Team/Channel members only, and non-members can't see or access the chat thread. If IB is enabled for the tenant and a non-team member attempts to join a channel meeting, that user isn't allowed to join the meeting. However, if IB is _not_ enabled for the tenant and a non-team member attempts to join a channel meeting, the user is allowed to join the meeting—but they won't see the chat option in the meeting.
 
+- **Maximum number of segments allowed in a tenant**: Each tenant can set up to 100 segments for configuring IB policies. Although, there is no limit on number of policies that can be configured.
```
Comment (JohanFreelancer9) [inline Teams/information-barriers-in-teams.md:200]:
```suggestion
- **Maximum number of segments allowed in a tenant**: Each tenant can set up to 100 segments for configuring IB policies. However, there is no limit on the number of policies that can be configured.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@chrfox PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by anwara-ms** [comment]
+ Rob as FYI

Thank you!
 --
Anwar Ali | PM @ TEAMS | LinkedIn<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.linkedin.com%2Fin%2Fanwarali&data=04%7C01%7CAli.Anwar%40microsoft.com%7C61a24ee754db4b1493e408d6b4dc7b41%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636895256392090042%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C-1&sdata=KpQ4z1jxZOc8s8IWXAYS3oVAbqKpT94E%2BzEB%2FLypv2c%3D&reserved=0>
IP, Security and Compliance Team

**Comment by robmazz** [comment]
@anwara-ms - Instead of managing the conflict in here, I'm going to close this PR and make the change on private repo and publish. I'll also reach out to you today about how we should manage updates moving forward. Thanks!

**Pull request closed without merging**